src/main/java/com/iota/iri/conf/ProtocolConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.iota.iri.conf;
+
+public interface ProtocolConfig extends Config {
+
+    int getMwm();
+
+    int getTransactionPacketSize();
+
+    int getRequestHashSize();
+
+    double getpReplyRandomTip();
+
+    double getPDropTransaction();
+
+    double getpSelectMilestoneChild();
+
+    double getPSendMilestone();
+
+    double getPPropagateRequest();
+
+    int getMinimumWeightMagnitude();
+
+    interface Descriptions {
+        String MWM = "The minimum weight magnitude is the number of trailing 0s that must appear in the end of a transaction hash. Increasing this number by 1 will result in proof of work that is 3 times as hard.";
+        String TRANSACTION_PACKET_SIZE = "The size of the packet in bytes recieved by a node. In the mainnet the packet size should always be 1650. It consists of 1604 bytes of a received transaction and 46 bytes of a requested transaction hash. This value can be changed in order to create testnets with different rules.";
+        String REQUEST_HASH_SIZE = "The size of the requested hash in a packet. It's size is derived from the minimal MWM value the network accepts. The larger the MWM -> the more trailing zeroes we can ignore -> smaller hash size.";
```

Comment:
I'm being a grammar nazi here :trollface: `It's` should be `Its`

Reply:
:+1: